jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -276,11 +276,10 @@ def delete_subsets(self):
         Delete all subsets app-wide.  Required before changing ``align_by``.
         """
         # subsets will be deleted on changing link type:
-        for subset_group in self.app.data_collection.subset_groups:
-            self.app.data_collection.remove_subset_group(subset_group)
+        self.app.delete_subsets()
 
     def vue_delete_subsets(self, *args):
-        self.delete_subsets()
+        self.app.delete_subsets()

```

Comment:
this one might be useful to leave in case we ever add additional checks, etc, in the plugin method